06week/dodgeball.js

```diff
@@ -0,0 +1,202 @@
+  },
+  {
+    id: 7,
+    name: "Mia Watu",
+    age: 17,
+    skillSet: "acrobatics",
+    placeBorn: "Los Angeles, California"
+  },
+  {
+    id: 8,
+    name: "Walter Cole",
+    age: 32,
+    skillSet: "jump rope",
+    placeBorn: "New Orleans, Louisiana"
+  },
+]
+//take list of people and create new player instances
+//there should NOT be two teammate classes only one not blueTeammate and redTeammate
+//teammate class which represents a player once they get selected for a team
+//attribute of teammate class should be teamname all teammates have an attribute called team name
+//if you blue your attribute will say blue if youre red your attribute will say red
+const listOfPlayers = []
+const blueTeam = []
+const redTeam = []
+
+class player {
+  constructor(id, name, age, skillSet, placeBorn){
+    this.id = id;
+    this.name = name;
+    this.age = age;
+    this.skillSet = skillSet;
+    this.placeBorn = placeBorn;
+
+    //listPeopleChoices
+  }
+}
+
+//the parent class is player, the child is teammate
+class Teammate extends player {
+  constructor(id, name, age, skillSet, placeBorn, teamName){
+    super(id, name, age, skillSet, placeBorn);//super calls player constructor
+      this.teamName = teamName;
+    /*if your a blue teammate you attribute will say blue,
+    if you a red teammamte your attribute will say red*/
+    }
+}
+//teammate class should extend the player class
+//Ex: truck extends vehicle
+//every teammate has the same attributes of player plus one more
+//which is which team theyre on 
+const listPeopleChoices = () => {
+  const listElement = document.getElementById('people')
+  arrOfPeople.map(person => {
+    //loops through arrOfPeople array
+    //creates list item tag
+    const li = document.createElement("li")
+    //creates button element
+    const button = document.createElement("button")
+    //text in button reads make player
+    button.innerHTML = "Make Player"
+    //when the button is clicked...
+    button.addEventListener('click', function() {makePlayer(person.id)} )
+    li.appendChild(button)
+    li.appendChild(document.createTextNode(person.name + " - " + person.skillSet))
+    listElement.append(li)
+  })
+}
+
+const makePlayer = (id) => {
+  let playerElement = arrOfPeople.find(function(player){
+    //find players with "id" in arrOfPeople
+    return player.id == id;
+
+});
+  //goes through index of arrOfPeople with id
+  let index = arrOfPeople.indexOf(playerElement);
+  //takes out selected value 
+  arrOfPeople.splice(index, 1);
+  //removes 1 item from arrayOfPeople array
+  let newPlayer = new player (playerElement.id, playerElement.name, playerElement.age, playerElement.skillSet, playerElement.placeBorn);
+  //creates a new player object
+  //adds newPlayer object to listOfPlayers array
+  listOfPlayers.push(newPlayer);
+  //calls PlayerDisplay function from below
+  //PlayerDisplay();
+  console.log(newPlayer)
+  PlayerDisplay();
+  };
+
+const makeBluePlayer = (player) => {
+
```

Comment:
This would have been the place to create a Teammate instance and have it store the team color.